.travis.yml

```diff
@@ -4,7 +4,7 @@ jdk:
 - oraclejdk8
 before_install:
 - chmod +x ./gradlew
-- chmod +x ./BuildDevint
+- chmod +x ./BuildDevint_travis
```

Comment:
Another solution is to add a new "-d <eclipse/ij2016/ij2017>" argument on BuildDevint script to control disable which build steps.  And open this switch on .travis.yml file.

See code snippet:  `while getopts ":hqvd:" option; do`